src/ansys/dpf/core/operators/math/modal_damping_ratio.py

```diff
@@ -179,7 +179,7 @@ class InputsModalDampingRatio(_Inputs):
     --------
     >>> from ansys.dpf import core as dpf
     >>> op = dpf.operators.math.modal_damping_ratio()
-    >>> my_natural_freq = dpf.FieldsContainer()
+    >>> my_natural_freq = dpf.()
```

Comment:
This is not ok

Reply:
@PProfizi This was originally a bug (the op was not accepting a FC, but a vector of doubles).